portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,8 +35,8 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   
-  private List<String> output = new ArrayList<>();
-  
+  private Comments comments = new Comments(5, new ArrayList<String>());
```

Comment:
"5" can be defined as a constant, e.g. private static final int COMMENT_LIMIT = 5.